_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -245,10 +245,10 @@ class HealthCareApp extends React.Component {
 
     return (
       <div className="row">
-        <div className="small-4 columns">
+        <div className="medium-4 columns show-for-medium-up">
           <Nav currentUrl={this.props.location.pathname}/>
```

Comment:
We don't need `currentUrl` anywhere.  `this.props.location` seem to be available in all components.